src/index.html

```diff
@@ -4,9 +4,384 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#HomePage">
+        <img
+          class="logo__img"
+          src="./images/logo.png"
+          alt="logo MOYO"
+        >
+      </a>
+
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link is-active" href="#Apple">
+              Apple
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">
+              samsung
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#Mobile">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#PC"
+              data-qa="nav-hover"
+            >
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#Gadjets">
+              Гаджеты
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#Tablet">
+              Планшеты
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#Photo">
+              Фото
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#Video">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <ul class="catalog__list">
+          <li class="catalog__item">
+            <div class="product-card" data-qa="card">
+              <img
+                class="product-card__image"
+                src="./images/imac.jpeg"
+                alt="APPLE 27"
+              >
+              <h2 class="product-card__title">
```

Comment:
add one empty line between same level elements
```suggestion
              >
              
              <h2 class="product-card__title">
```